apps/nest-server/src/app/app.service.ts

```diff
@@ -48,7 +48,14 @@ export class AppService {
       const results = await client.query(
         `select returnflag, linestatus, sum(quantity) as sum_qty, sum(extendedprice) as sum_base_price, sum(extendedprice * (1 - discount)) as sum_disc_price, sum(extendedprice * (1 - discount) * (1 + tax)) as sum_charge, avg(quantity) as avg_qty, avg(extendedprice) as avg_price, avg(discount) as avg_disc, count(*) as count_order from lineitem where shipdate <= date '1998-12-01' group by returnflag, linestatus order by returnflag, linestatus`,
       )
-      return { columns: results.columns, rows: results.data }
+      return {
+        columns: results.columns,
+        rows: JSON.stringify(results.data, (key, value) => {
+          if (typeof value !== 'bigint') return value
+
+          return value.toString()
```

Comment:
Can u please add to the README this limitation?

Reply:
already did